Source/WebCore/platform/EventTrackingRegions.h

```diff
@@ -39,23 +40,71 @@ enum class TrackingType : uint8_t {
 };
 
 struct EventTrackingRegions {
+    enum class Event : uint8_t {
+        MousedownEvent,
```

Comment:
Given that this is already an enum class, I think you can drop the `Event` suffix for each entry e.g. just use `MouseDown` instead of `MousedownEvent`.  Every client will be referencing `Event::MouseDown` anyway.  There's no information loss with dropping the suffix, and it makes the code more compact.

Reply:
I'll drop `Event` postfix. But since this is aligned to DOM's event names, we keep current `mousedown` etc.